pkgs/games/eternity-engine/default.nix

```diff
@@ -1,29 +1,29 @@
-{ lib, stdenv, cmake, libGL, SDL, SDL_mixer, SDL_net, fetchFromGitHub, makeWrapper }:
+{ lib, stdenv, cmake, libGL, SDL2, SDL2_mixer, SDL2_net, fetchFromGitHub, makeWrapper }:
 
 stdenv.mkDerivation rec {
   pname = "eternity-engine";
-  version = "3.42.02";
+  version = "4.02.00";
   src = fetchFromGitHub {
     owner = "team-eternity";
     repo = "eternity";
     rev = version;
-    sha256 = "00kpq4k23hjmzjaymw3sdda7mqk8fjq6dzf7fmdal9fm7lfmj41k";
+    fetchSubmodules = true;
```

Comment:
IT would be great if we could unvendor https://github.com/Wohlstand/libADLMIDI/-

Reply:
How so? Am new to packaging for Nix, so I don't understand much. Do you mean creating a nixpkg for libADLMIDI?

Reply:
> What do you mean by unvendor? Am new to packaging for Nix, so I don't understand much. Do you mean creating a nixpkg for libADLMIDI?

Removing the submodule and creating a package for the programs that where a submodule or "vendored".

The goal is that we have one package for each software/library used which could theoretically be reused be another project and is development independently of the project. This helps us by not duplicating the software and building it multiple times and keeping it patched.